test/bdd/steps/network.js

```diff
@@ -14,6 +14,7 @@ const OtNode = require('./lib/otnode');
 const Utilities = require('../../../modules/Utilities');
 const LocalBlockchain = require('./lib/local-blockchain');
 const httpApiHelper = require('./lib/http-api-helper');
+const ImportUtilities = require('../../../modules/ImportUtilities');
```

Comment:
Do not use the code outside the the bdd dir. That way you are testing the code with itself.